Task Liverpool/src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage extends BasePage {
+
+    private By searchBar =By.id("mainSearchbar");
+    private By searchBarButton = By.cssSelector(".input-group-text .icon-zoom");
+
+    public HomePage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public void clickSearchBar(){
+        click(searchBar);
+    }
+
+    public void typeSearchBar(){
+        type(searchBar, "1103656130");
+    }
+
+    public void clickSearchBarButton(){
```

Comment:
for me is complicated to difference between `clickSearchBar` and `clickSearchBarButton`, do we really need the first one? for which scenarios? maybe I recomment to have a single method `findProduct(String product)` and inside of it do de click and type on that element

Reply:
Well, usually the user clicks first on the Search Bar before typing something. I wanted the flow to be as close as the real one as possible.